documentation/updates.md

```diff
@@ -7,34 +7,48 @@ redirect_from:
 
-You need to backup both: your **database** and your file system, especially the file **includes/autoconf.php**.
+## Step 2 - Download and unzip
+Change to the folder were you will be installing the new version of Kimai and download the newest version with something like:
+```
+wget https://github.com/kimai/kimai/releases/download/1.3.1/kimai_1.3.1.zip
+```
+The link to the newest version can be found on the [Downloads](/download/) page.
 
-## Step 2
+Unzip it with something like this:
+```
+unzip kimai_1.3.1.zip
+```
+## Step 3 - copy configuration
 
-To install a new version just replace the entire directory with the new package and **KEEP** the file **autoconf.php** which can be found inside the /includes folder.
+Copy `autoconfig.php` from the old version to the new version of Kimai with something like this:
+```
+cp public_html-backup/includes/autoconfig.php public_html/includes/
+```
 
-Additionally you have to set write permissions to the **temporary/** folder.
+## Step 4 - Set correct permissions
 
-Attention: Kimai will NOT work with wrong permissions!
+Insure that `temporary` folder has write permissions.
 
-For Linux that could be:
+**Attention: The Kimai Updater will NOT work with wrong permissions!**
 
+You can set write permissions to the 'temporary' folder with:
+```
+chmod -R og+w temporary/
+```
+This will give write permissons to the owner and group. The best is to only give the write permissions to PHP and the webserver in this way instead of giving all users write permissions. This can be or is a risk, especially in shared hosting environments.
+If setting the permissions with the command above does not work you can give every user write permissions with:
 ```
 chmod -R 777 temporary/
 ```
-It would be even better, if you give writing permission to your PHP or Webserver user only, instead of making it world 
-writable. The last can be or is a risk, especially in shared hosting environments.
-
-## Step 3
+## Step 5 - Run the updater.php
 
 Ready for update:
 
-Visit your Kimai installation with your web browser and the auto update process will perform all necessary tasks
-(eg. http://kimai.example.com/ - do **NOT** execute the installer!).
+Visit your Kimai installation (eg. http://kimai.example.com/) with your web browser and the auto update process will perform all necessary tasks. You should see the **updater**. If you see the **installer** do not continue. Check that you have copied the `includes/autoconfig.php` from your old installation to the new. Do **NOT** execute the installer, as doing so will misconfigure the new installation and you will need to restore the backup of the database and start the update process again.
```

Comment:
Check that you have copied the **file** `includes/autoconfig.php`

Reply:
Commited